kubernetes/provider_config.go

```
@@ -0,0 +1,256 @@
+				return nil, err
+			}
+			log.Printf("[DEBUG] Using kubeconfig: %s", path)
+			expandedPaths = append(expandedPaths, path)
+		}
+		if len(expandedPaths) == 1 {
+			loader.ExplicitPath = expandedPaths[0]
+		} else {
+			loader.Precedence = expandedPaths
+		}
+
+		ctxSuffix := "; default context"
```

golangci-lint (2.12.2)

[error] ineffectual assignment to ctxSuffix

(ineffassign)